### ISSUE TITLE
MySQLException: Unknown system variable 'query_cache_size'

### ISSUE THREAD

**Issue by facundofarias** [metabase/metabase#6984]
Hi there, 

I am trying to connect metabase docker container with another container that is running MySQL (MySQL 8.0.3), which is the one that provides [laradock](http://laradock.io/).
Then, I get the following error message:
```
02-20 13:11:56 ERROR metabase.driver :: Failed to connect to database: java.sql.SQLException: Unknown system variable 'query_cache_size'
02-20 13:11:56 DEBUG metabase.middleware :: POST /api/setup/validate 400 (108 ms) (0 DB calls)
{:errors {:dbname "java.sql.SQLException: Unknown system variable 'query_cache_size'"}}
```
As can be seen on the [docs](https://dev.mysql.com/doc/refman/5.7/en/query-cache.html), the query_cache_size was removed on MySQL8. JetBrains guys have suffered the same [issue](https://youtrack.jetbrains.com/issue/DBE-5212), is seems to be that it is fixed by updating the driver to the version MySQL JDBC driver v 5.1.44.

Thanks!

**Comment by facundofarias** [comment]
I've published the fix on [docker hub](https://hub.docker.com/r/nestednet/metabase), and now it's working with MySQL 8.0.3.

**Comment by salsakran** [comment]
fixed via #6990
